lib/src/style/url.dart

```diff
@@ -63,8 +63,7 @@ class UrlStyle extends InternalStyle {
     }
 
     var index = path.indexOf("/");
-    if (index > 0 && path.startsWith('://', index - 1)) {
-    }
+    if (index > 0 && path.startsWith('://', index - 1)) {}
```

Comment:
@nex3 – fun when dartfmt finds these types of things...

Reply:
You can just get rid of this.